bibliopixel/remote/control.py

```diff
@@ -48,8 +58,18 @@ def __init__(self, layout, animations,
             adesc['data']['display'] = adesc['data'].get('display', adesc['name'])
             adesc['name'] = normalize_name(adesc['name'])
             anim['run'] = anim.get('run', {})
+            if auto_demo:
```

Comment:
The fact that you keep testing this boolean to see what to do is a code smell - it means that you have another class there waiting to get out.